tck/src/main/java/org/eclipse/microprofile/concurrency/tck/TckTest.java

```diff
@@ -30,14 +42,94 @@
 
     @Deployment
     public static WebArchive createDeployment() {
+        // build a JAR that provides fake 'ThreadPriority' context type
+        JavaArchive threadPriorityContextProvider = ShrinkWrap.create(JavaArchive.class, "threadPriorityContext.jar")
+                .addPackage(ThreadPriorityContextProvider.class.getPackage().getName())
```

Comment:
I would rather we hardcode the package name here instead of getting it from a class.  Looking at this code I don't know what package we are pulling in, and I'd need to hover over it in an IDE or expand imports to figure out what package we are pulling in.
Also, note that there is an `addPackages(boolean recursive, String... pkgNames)` which you may find useful.

Reply:
That makes sense, and I like the idea of using recursive to avoid needing to list all of the packages. I'll update it.